src/main/java/net/minecraftforge/common/extensions/ForgeRecipeManager.java

```diff
@@ -72,6 +100,7 @@ public ItemStack getResult(IInventory input, World world, RecipeType<?> type)
     /**
      * Modders should not use this if possible.  In the context that this is used, you should already have the recipe.  Do not run extra lookups using this method.
      */
+    @Deprecated
```

Comment:
Why?

Reply:
Same reason the comment is there.  Method callers should try to reduce recipe lookups as much as possible, this method does not support that.  Nobody should be using it.

Reply:
So call the `getRemaingItems` for the recipe object instead. Good suggestion; I hope you can document it.

Reply:
Noted to call IRecipe#getRemainingItems in javadoc of that method.